tests/utils.py

```diff
@@ -187,7 +187,7 @@ def _naive_mask_apply(masks, data):
     for mask in masks:
         assert mask.shape == data.shape[2:], "mask doesn't fit frame size"
 
-    dtype = np.result_type(*masks, data)
+    dtype = np.result_type(*[m.dtype for m in masks], data.dtype)
```

Comment:
Is this needed for sparse arrays?

Reply:
Yes, for some reason result_type doesn't accept sparse arrays anymore with numpy 1.17